test/manifests/normal/create.yaml

```diff
@@ -1,16 +1,16 @@
-name: $RESOURCEGROUP
-location: $AZURE_REGION
+name: "{{ .Env.RESOURCEGROUP }}"
```

Comment:
@kargakis @asalkeld I don't like the fact that this has changed to a Go template, I wish I'd seen this before it merged :(

Reply:
@kargakis @asalkeld I get the feeling there is some over-engineering going on here and we may need to refocus.

Reply:
@jim-minter what's the difference between this and the previous form? If we are only going to use these manifests in our dev and CI environments then what's the point?

Reply:
The benefit with keeping this in Go (among other changes) is less dependance on the host (eg. for replacing the envs in the manifest before, we had to install `gettext` in the test base image and get to use `envsubst`. Just an example. Others include the hack/dns and hack/aad rewrites to remove our dependance on `az`.